internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -659,3 +655,130 @@ func RestoreIrqBalanceConfig(ctx context.Context, irqBalanceConfigFile, irqBanne
 	}
 	return nil
 }
+
+// convertAnnotationToLatency converts the cpu-c-states.crio.io annotation to a maximum
+// latency value in microseconds.
+//
+// The cpu-c-states.crio.io annotation can be used to control c-states in several ways:
+//
+//	enable: enable all c-states (cpu-c-states.crio.io: "enable")
+//	disable: disable all c-states (cpu-c-states.crio.io: "disable")
+//	max_latency: enable c-states with a maximum latency in microseconds
+//	             (for example,  cpu-c-states.crio.io: "max_latency:10")
+//	cstate.id: enable c-states with a maximum kernel CPU idle level ID
+//	           (for example,  cpu-c-states.crio.io: "cstate.id:1")
+//	cstate.name: enable c-states using a case sensitive name of the maximum C-state allowed
```

Comment:
just a nit, you're using "C-state" and "c-state" interchangeably in this section. I'd stick with one, and lean towards "c-state"

Reply:
Thanks Tom - I will address this nit if I upload an additional set of changes.

Reply:
I made this change when I did the rebase just now.